src/kit.ts

```diff
@@ -603,7 +603,7 @@ const MSVC_ENVIRONMENT_VARIABLES = [
   'LIBPATH',
   'NETFXSDKDir',
   'Path',
-  'Platform',
+  //'Platform',
```

Comment:
I don't actually see Platform ever set in any of the Developer Command prompts I've opened (VS 2017-VS 2022).  We should just delete this line.

cc: @lygstate

Reply:
Platform  is set by VS2019, maybe we need a better way to detecting if Platform are set?